ts/getGossipPeers.ts

```diff
@@ -12,16 +12,16 @@ const { shuffle } = require('./utils')
  * @returns {Set<Peer>}
  *
  */
-module.exports = function getGossipPeers (router, topic, count) {
+export function getGossipPeers (router: Gossipsub, topic: string, count: number): Set<Peer> {
```

Comment:
This is not equal to original.
`module.exports =  function getGossipPeers` should be exported as  `export = function getGossipPeers` to not introduce breaking change if somebody is using this directly

Reply:
I'm going to leave this one as is. Its not part of the exported interface, so there's not any expectation of backwards compatibility here.